core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -642,6 +642,7 @@ public void start() {
       if (globalConfiguration.security().authorization().enabled()) {
          globalConfiguration.security().authorization().principalRoleMapper().setContext(new PrincipalRoleMapperContextImpl(this));
       }
+      globalComponentRegistry.start();
```

Comment:
Also this change breaks a few tests.

# CacheManagerTest.testExceptionOnCacheManagerStop - simple fix
# CacheManagerNotifierTest.testMockViewChange - personally think this one might as well remove